x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -223,7 +223,8 @@ public static NamedClusterPrivilege resolve(String name) {
     public static boolean isClusterAction(String actionName) {
         return actionName.startsWith("cluster:") ||
             actionName.startsWith("indices:admin/template/") ||
-            actionName.startsWith("indices:admin/index_template/");
+            actionName.startsWith("indices:admin/index_template/") ||
+            actionName.startsWith("indices:admin/xpack/rollup");
```

Comment:
I'm thinking the right thing to do here is to leave the MANAGE_ROLLUP as is, and instead introduce a new MANAGE_ROLLUP index privilege

Reply:
not sure what the proper mental model here is, since rollup jobs were originally cluster privileges that created new indices